### ISSUE TITLE
Fatal error is displayed after enabling media in group

### ISSUE THREAD

**Issue by pavanpatil1** [rtCamp/rtMedia#1854]
```
Enviornment: 
WordPress: v5.9.1
BuddyPress: v10.1.0
rtMedia: v4.6.10
BP rewrites: v1.0.0-beta2
```

URL: <site-name>/groups/<group-name>

After enabling media in the group the fatal error is displayed on the group page related to the navbar and the page is not accessible. 

**Steps to reproduce:** 
1. Install and activate the [BP rewrites](https://bpdevel.wordpress.com/2022/03/12/bp-rewrites-1-0-0-beta2/) plugin. 
2. From the backend go to rtMedia -> BuddyPress and enable the group in media option and navigate to the group page and observe the issue. 

https://user-images.githubusercontent.com/44057535/158294989-2131bd79-8b24-44e7-a73d-219c6a7b1b64.mp4

**Comment by vishalkakadiya** [comment]
@thebengalboy This issue will be fixed with the `feature/bprewrite` branch, but we can't directly merge this to `pre-release` as some other things are still not working in this, once we fixed all then we can merge this to `pre-release`. Thanks!
CC @Pradeep1308

**Comment by pavanpatil1** [comment]
Closing this issue as fixed in #1887
